README.md

```diff
@@ -8,7 +8,12 @@ sample branch changes
 
 Add your name and and one other line here:
 
+<<<<<<< HEAD
```

Comment:
@dominikkawka looks like there are some conflict stuff added here. Can you remove this